CMakeLists.txt

```diff
@@ -0,0 +1,308 @@
+cmake_minimum_required(VERSION 2.6)
+
+project("openscap")
+set(OPENSCAP_VERSION_MAJOR "1")
+set(OPENSCAP_VERSION_MINOR "3")
+set(OPENSCAP_VERSION_PATCH "0")
+set(OPENSCAP_VERSION "${OPENSCAP_VERSION_MAJOR}.${OPENSCAP_VERSION_MINOR}.${OPENSCAP_VERSION_PATCH}")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/")
+
+if(CMAKE_BUILD_TYPE STREQUAL "Release" OR CMAKE_BUILD_TYPE STREQUAL "RelWithDebInfo")
+	set(NDEBUG 1)
+endif()
+
+# ---------- INCLUDES CMAKE MODULES
+
+include(GNUInstallDirs)
+include(FindPkgConfig)
+include(CheckLibraryExists)
+include(CheckFunctionExists)
+include(CheckIncludeFile)
+include(CheckCSourceCompiles)
+include(CPack)
+
+# ---------- DEPENDENCIES
+
+# ACL
+find_package(ACL)
+if(ACL_FOUND)
+	CHECK_LIBRARY_EXISTS(acl acl_extended_file "" HAVE_ACL_EXTENDED_FILE)
+	CHECK_INCLUDE_FILE(acl/libacl.h HAVE_ACL_LIBACL_H)
+	CHECK_INCLUDE_FILE(sys/acl.h HAVE_SYS_ACL_H)
```

Comment:
code style

Reply:
@mpreisler What code style is expected?

Reply:
lower case macro and function calls, upper case variables. that's what we use in SSG and workbench